lib/pbench/test/unit/agent/conftest.py

```diff
@@ -65,3 +66,11 @@ def setup(tmp_path_factory, agent_setup, monkeypatch) -> Dict[str, Path]:
         "_PBENCH_AGENT_CONFIG", str(agent_setup["cfg_dir"] / "pbench-agent.cfg")
     )
     return agent_setup
+
+
+@pytest.fixture(scope="session")
+def agent_logger():
```

Comment:
This could be `make_logger` like the server side, which would collapse a lot of the calls which `black` has exploded into multiple lines as it appears that one character difference between `self.logger` and `agent_logger` was the straw that broke black's back. Which isn't a big deal, but ...